streaming/src/test/scala/org/apache/spark/streaming/StreamingListenerSuite.scala

```diff
@@ -235,6 +236,19 @@ class StreamingListenerSuite extends TestSuiteBase with LocalStreamingContext wi
     verifyNoMoreInteractions(streamingListener)
   }
 
+  test("SPARK-38498: Support customized streaming listener") {
```

Comment:
`customized` -> `extra` because the conf name is `STREAMING_EXTRA_LISTENERS`.

Reply:
Done